src/test/java/com/exasol/dbbuilder/dialects/exasol/ExasolSchemaTest.java

```diff
@@ -36,6 +36,14 @@ void testCreateTableBuilder() {
         assertThat(table.getName(), equalTo("TABLE_D"));
     }
 
+    @Test
+    void testCreateAdapterScriptBuilder() {
```

Comment:
The name of this method does not fit the test inside.

Reply:
why not? It tests the method `exasolSchema.createAdapterScriptBuilder()`